main.c

```diff
@@ -160,6 +159,8 @@ int WINAPI _tWinMain (HINSTANCE hThisInstance,
   PrintDebug(_T("Shell32.dll version: 0x%lx"), shell32_version);
 #endif
 
+  /* Parse command-line options */
+  ProcessCommandLine(&o, GetCommandLine());
```

Comment:
On Sun, Aug 28, 2016 at 5:48 PM, ValdikSS <notifications@github.com> wrote:

> In main.c
> <https://github.com/OpenVPN/openvpn-gui/pull/76#discussion_r76539811>:
>
> > @@ -160,6 +159,8 @@ int WINAPI _tWinMain (HINSTANCE hThisInstance,
> >    PrintDebug(_T("Shell32.dll version: 0x%lx"), shell32_version);
> >  #endif
> >
> > +  /* Parse command-line options */
> > +  ProcessCommandLine(&o, GetCommandLine());
>
> I'm not sure if this absolutely correct to move this function. Can
> somebody please confirm it won't break anything?
>
The only requirement I can think of is that default options should be set
first, then the  registry scanned for user-preferences and only then
command-line options should be set so that the latter over-rides the
former, not the other way. I haven't looked at this patch to see whether
that order is preserved. If its necessary to parse some command-line
options early, some changes to the parsing code (like call it twice) may be
required.

Selva